.github/actions/release/changelog-generator/action.yml

```
@@ -0,0 +1,50 @@
+name: 'changelog-generator'
+description: 'Runs github changelog generator'
+inputs:
+  changelog-path:
+    description: "path to the changelog file"
+    required: false
+    default: ./CHANGELOG.md
+  changelog-config-path:
+    description: "path to the changelog config"
+    required: true
+  future-release:
+    description: "release name"
+    required: true
```

Is future release the version? Can we add an example?

```suggestion
  future-release:
    description: "release name (e.g. `v1.2.3-dev.1`)"
    required: true
```